### PR TITLE
Add rpm-ostree OCI-TA Task variant

### PR DESCRIPTION
**Pull request by zregvart**
Generated for version 0.2 only.

Reference: https://issues.redhat.com/browse/EC-924

### CODE REVIEW

File: task/rpm-ostree-oci-ta/0.2/rpm-ostree-oci-ta.yaml [hunk 1]
```diff
@@ -0,0 +1,303 @@
...
+            --tls-verify="${TLSVERIFY}" \
+            --digestfile "/var/workdir/source/image-digest" "${IMAGE}" \
+            "docker://${IMAGE}" && break || status=$?
+        done
+        if [ "$status" -ne 0 ]; then
+            echo "Failed to push sbom image to registry after ${max_run} tries"
+            exit 1
+        fi
+
+                                                                                 <"/var/workdir/source"/image-digest tee "$(results.IMAGE_DIGEST.path)"
```
Comment (zregvart) [inline task/rpm-ostree-oci-ta/0.2/rpm-ostree-oci-ta.yaml:282]: Something in the shell formatter does this, it's a generated file and this is mostly esthetics so I don't think it is worth troubleshooting, but yell at me if you think it matters...

**Comment by cgwalters** [comment]
Note we have https://issues.redhat.com/browse/BIFROST-408 which would allow us to get rid of this task entirely.

What's the timeframe for this? I would say it's possible within a few weeks we'd gain sufficient knowledge to say whether or not we can execute on deprecating and eventually removing this task entirely.

File: task/rpm-ostree-oci-ta/0.2/rpm-ostree-oci-ta.yaml (hunk 1 above)
Comment (tkdchen) [reply task/rpm-ostree-oci-ta/0.2/rpm-ostree-oci-ta.yaml:282]: Is shfmt the tool used?

Comment (zregvart) [reply task/rpm-ostree-oci-ta/0.2/rpm-ostree-oci-ta.yaml:282]: We use https://github.com/mvdan/sh

**Comment by lcarva** [comment]
To avoid blocking different efforts, let's go ahead and get this Task in. We can remove/deprecate it if/when we do so for the `rpm-ostree` Task as well.

**Comment by zregvart** [comment]
Taking the comments above as approval

**Review by lcarva — APPROVED** [review]

**Pull request merged**